roles/custom/matrix-media-repo/tasks/setup_install.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+- name: Ensure media-repo paths exist
+  ansible.builtin.file:
+    path: "{{ item.path }}"
+    state: directory
+    mode: 0750
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_groupname }}"
+  with_items:
+    - path: "{{ matrix_media_repo_base_path }}"
+      when: true
+    - path: "{{ matrix_media_repo_docker_src_files_path }}"
+      when: "{{ matrix_media_repo_container_image_self_build }}"
+  when: "item.when | bool"
+
+- name: Ensure media-repo configuration installed
+  ansible.builtin.template:
+    src: "{{ role_path }}/templates/media-repo/media-repo.yaml.j2"
+    dest: "{{ matrix_media_repo_base_path }}/media-repo.yaml"
```

Comment:
In other roles, we also try to keep the configuration:

- living in `config/`, separate from the data (living in `data/`)
- mounted into the container as read-only, if possible

Looking at https://github.com/turt2live/matrix-media-repo/blob/b7160248830520d5d3fbc401dbaa702b5e81696b/Dockerfile#L32 it looks like we may be able to:

- introduce a new `matrix_media_repo_config_path` variable (`matrix_media_repo_config_path: "{{ matrix_media_repo_base_path }}/config`) and create this new directory in the `Ensure media-repo paths exist` task above
- adjust this task to save the `media-repo.yaml` file in `matrix_media_repo_config_path`
- create a new `env` file in `{{ matrix_media_repo_base_path }}/env` which contains `REPO_CONFIG=/config/media-repo.yaml`. See `./roles/custom/matrix-sliding-sync/templates/env.j2` for an example
- add `--env-file={{ matrix_media_repo_base_path }}/env` to the systemd service
- add `--mount` for mounting `matrix_media_repo_config_path` to `/config` with `ro` (readonly)